lib/rides/pages/ride_detail_page.dart

```diff
@@ -152,13 +153,24 @@ class _RideDetailPageState extends State<RideDetailPage> {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageRideDetailTitle),
-        actions: <Widget>[
-          IconButton(
-            onPressed: () {},
-            icon: const Icon(Icons.chat),
-            tooltip: S.of(context).openChat,
-          )
-        ],
+        actions: _fullyLoaded
+            ? <Widget>[
+                IconButton(
```

Comment:
Just noticed that this one does not have a badge. We probably should add that.

Reply:
thanks for the hint